libraries/api/discussion_helper.cpp

```diff
@@ -193,7 +254,9 @@ namespace golos { namespace api {
                         }
                     }
 
-                    if (db.has_hardfork(STEEMIT_HARDFORK_0_19__898)) {
+                    if (db.has_hardfork(STEEMIT_HARDFORK_0_19__898) &&
+                        d.auction_window_reward_destination == protocol::to_reward_fund
```

Comment:
What about case when all votes in auction window? You're lost such rewards.

Reply:
Yes, you're right. Fixed that case.

Reply:
Here should only checking on HF.
If exist any unclaimed_reward it should be returned to reward fund after HF19